.github/workflows/ci.yml

```diff
@@ -210,6 +210,8 @@ jobs:
           kubectl delete pvc --all-namespaces --all
           kubectl delete backupstorage --all-namespaces --all
           kubectl get ns -o name | grep kuttl  | awk -F '/' {'print $2'} | xargs --no-run-if-empty kubectl delete ns
+          kubectl delete ns operators olm || true
```

Comment:
These namespaces are leftovers from previous releases, I don't think we need use them at all right now. Are these getting provisioned anywhere? Can't we do this cleanup manually just one time, instead?

Reply:
There's a test step in the end to do a clean-up, but that happens only if the test succeeds. I added this in the clean-up step separately so that they're cleaned up even in case of failures